.cirrus.yml

```diff
@@ -7,8 +7,11 @@ fedora_30_task:
   env:
     PYTHON: python3
     matrix:
+      # Get deps from setup.py (therefor pip version of avocado)
```

Comment:
Fun fact: my dictionary says that "therefor" is an "archaic" word, while "therefore" is the "modern" one.

Reply:
Therefor is 1 char shorter, therefor it's better for the environment (-108 chars to describe the benefit :D)

Reply:
:D